src/script/src/python/ffi_types/vector/tests.rs

```diff
@@ -155,8 +155,10 @@ fn eval_pyo3(testcase: TestCase, locals: HashMap<String, PyVector>) {
 }
 
 fn eval_rspy(testcase: TestCase, locals: HashMap<String, PyVector>) {
-    vm::Interpreter::without_stdlib(Default::default()).enter(|vm| {
+    vm::Interpreter::with_init(Default::default(), |vm| {
```

Comment:
We don't want the stdlib, so does `with_init` create the interpreter with stdlib or not?

Reply:
> We don't want the stdlib, so does `with_init` create the interpreter with stdlib or not?

This wouldn't init stdlib unless explictly write `vm.add_native_modules` etc.